### DISCUSSION TITLE
Is there an options for disable alternative screen/buffer like iTerm2

### DISCUSSION THREAD

**Discussion by soomtong** [wezterm/wezterm#3869] [Q&A] [answered]
Hi, am moving wezterm from iTerm2.
I see pager/less working in alternative buffer. then quit paging/less, that screen disapears.

sometime it annoying for me. I disable this feature in iTerm2 like this.
Is an options like this in wezterm?

<img width="299" alt="image" src="https://github.com/wez/wezterm/assets/906974/4c868272-4f46-4ce3-bda0-5c5911ce0bcc">

**Comment by bew** [comment, marked as answer]
I don't think so..
However if you're using `less` as your main pager you could configure it to re-print its last visible screen of text (I use this some time to time, on demand with a keybind).

The less option is `--redraw-on-quit`, added around version 600 of less.
You can read more about that in https://github.com/gwsw/less/issues/36

**Reply by soomtong** [reply]
Oh, I see `less -XF` works as I hoped. I will use it for. thank you!
